### PR TITLE
feat: nightly npm releases

### PR DESCRIPTION
**Pull request by smrz2001**

### CODE REVIEW

File: .github/workflows/npm-nightly.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'Nightly branch'
+        required: false
+        default: 'develop'
+
+jobs:
+  publish:
+    uses: ./.github/workflows/reusable-release.yml
```
Comment (3benbox) [inline .github/workflows/npm-nightly.yml:15]: omg, what is this. TIL

File: .github/workflows/reusable-release.yml [hunk 2]
```diff
@@ -266,3 +266,52 @@ jobs:
...
+          git config user.email "${{ github.actor }}@users.noreply.github.com"
+          git config user.name ${{ github.actor }}
+      -
+        name: Install Dagger
+        uses: dagger/dagger-for-github@v3
+        with:
+          install-only: true
+          version: "0.2.36"
+      -
+        name: Verify Docker image
```
Comment (3benbox) [inline .github/workflows/reusable-release.yml:301]: I think it's fine to not verify a nightly, but if it's easy to do, no harm

**Review by 3benbox — APPROVED** [review]
LGTM

File: .github/workflows/npm-nightly.yml (hunk 1 above)
Comment (smrz2001) [reply .github/workflows/npm-nightly.yml:15]: Lol yeah, you can reuse GH workflows.

File: .github/workflows/reusable-release.yml (hunk 2 above)
Comment (smrz2001) [reply .github/workflows/reusable-release.yml:301]: Yes, super easy to do, and doesn't add too much time to the process either.

**Pull request merged**